puppet/modules/chassis/manifests/openssl.pp

```diff
@@ -0,0 +1,28 @@
+define chassis::openssl (
+) {
+	file { '/vagrant/certs':
+		ensure => 'directory',
+	}
+
+	# include ::openssl
+
+	# class { '::openssl':
+	  # package_ensure         => latest,
+	  # ca_certificates_ensure => latest,
+	# }
+	
+	$commonname = "*.${fqdn}"
+
+	openssl::certificate::x509 { $fqdn:
+	  country      => 'HM',
+	  organization => $fqdn,
+	  commonname   => $commonname,
+	} ->
```

Comment:
This should be specified as `require => Openssl::Certificate::X509[$fqdn]` on the file resource instead.